### PR TITLE
Added sorting to the labels & milestones page

### PR DESCRIPTION
**Pull request by Bwko**
Solves  #10

### CODE REVIEW

File: conf/locale/locale_en-US.ini [hunk 1]
```diff
@@ -604,6 +606,12 @@ milestones.edit_success = Changes of milestone '%s' has been saved successfully!
 milestones.deletion = Milestone Deletion
 milestones.deletion_desc = Deleting this milestone will remove its information in all related issues. Do you want to continue?
 milestones.deletion_success = Milestone has been deleted successfully!
+milestones.filter_sort.closestduedate = Closest due date
```
Comment (andreynering) [inline conf/locale/locale_en-US.ini]: Use underscore for translation keys:

`closestduedate` -> `closest_due_date`

**Review by andreynering — CHANGES_REQUESTED** [review]
Thank you for the PR.

Made just a single small request of change.

**Comment by Bwko** [comment]
@andreynering I've updated my PR

**Comment by andreynering** [comment]
LGTM

**Review by andreynering — APPROVED** [review]

**Comment by thibaultmeyer** [comment]
LGTM

File: templates/repo/issue/labels.tmpl [hunk 2]
```diff
@@ -33,7 +33,21 @@
 			</form>
 		</div>
 		<div class="ui divider"></div>
-
+        <div class="ui right floated secondary filter menu">
+            <!-- Sort -->
+			<div class="ui dropdown type jump item">
```
Comment (tboerger) [inline templates/repo/issue/labels.tmpl]: Please fix the indentation

**Review by tboerger — CHANGES_REQUESTED** [review]

**Comment by Bwko** [comment]
@tboerger Do you mean the` <!-- Sort --> & </div>?`
Is there a tool to automatically format the Golang template files?

**Comment by tboerger** [comment]
If your editor supports editorconfig it should save it properly

**Comment by Bwko** [comment]
My editor didn't support editorconfig files, I had to add a plugin.
Fixed & rebased

File: templates/repo/issue/milestones.tmpl [hunk 3]
```diff
@@ -22,7 +22,23 @@
...
+					<a class="{{if or (eq .SortType "closestduedate") (not .SortType)}}active{{end}} item" href="{{$.Link}}?sort=closestduedate&state={{$.State}}">{{.i18n.Tr "repo.milestones.filter_sort.closest_due_date"}}</a>
+					<a class="{{if eq .SortType "furthestduedate"}}active{{end}} item" href="{{$.Link}}?sort=furthestduedate&state={{$.State}}">{{.i18n.Tr "repo.milestones.filter_sort.furthest_due_date"}}</a>
+					<a class="{{if eq .SortType "leastcomplete"}}active{{end}} item" href="{{$.Link}}?sort=leastcomplete&state={{$.State}}">{{.i18n.Tr "repo.milestones.filter_sort.least_complete"}}</a>
+					<a class="{{if eq .SortType "mostcomplete"}}active{{end}} item" href="{{$.Link}}?sort=mostcomplete&state={{$.State}}">{{.i18n.Tr "repo.milestones.filter_sort.most_complete"}}</a>
+					<a class="{{if eq .SortType "mostissues"}}active{{end}} item" href="{{$.Link}}?sort=mostissues&state={{$.State}}">{{.i18n.Tr "repo.milestones.filter_sort.most_issues"}}</a>
+					<a class="{{if eq .SortType "leastissues"}}active{{end}} item" href="{{$.Link}}?sort=leastissues&state={{$.State}}">{{.i18n.Tr "repo.milestones.filter_sort.least_issues"}}</a>
+				</div>
+			</div>
+        </div>
```
Comment (tboerger) [inline templates/repo/issue/milestones.tmpl]: Still some wrong indentation :D

**Comment by Bwko** [comment]
Solved

**Comment by tboerger** [comment]
LGTM, will be merged when we branch the 1.0 release

**Review by tboerger — APPROVED** [review]

**Comment by lunny** [comment]
This maybe needs rebase and resolved the conflicts.

**Comment by Bwko** [comment]
@lunny I can't see the conflict. Only the conflicting file but not which line

**Comment by lunny** [comment]
You can handle the conflict before merging. You have to handle it on command line and your local.

**Comment by Bwko** [comment]
Fixed

**Comment by lunny** [comment]
Great, this PR will be delayed until 1.0 released.

**Comment by lunny** [comment]
Please resolve the conflict and we will merge it.

**Comment by lunny** [comment]
LGTM work

**Pull request merged**